src/app/components/projects/import-project-dialog/import-project-dialog.component.ts

```diff
@@ -138,7 +141,8 @@ export class ImportProjectDialogComponent implements OnInit {
   }
 
   prepareUploadPath(): string {
+    this.uuid = uuid();
     const projectName = this.projectNameForm.controls['projectName'].value;
-    return `http://${this.server.host}:${this.server.port}/v2/projects/${uuid()}/import?name=${projectName}`;
+    return `http://${this.server.host}:${this.server.port}/v2/projects/${this.uuid}/import?name=${projectName}`;
```

Comment:
Can we have configuration or common place of defining URI to server?

Reply:
Moved to project service